MimeKit/Cryptography/RsaEncryptionPadding.cs

```diff
@@ -182,8 +178,8 @@ public override string ToString ()
 		/// <returns><c>true</c> if <paramref name="left"/> and <paramref name="right"/> are equal; otherwise, <c>false</c>.</returns>
 		public static bool operator == (RsaEncryptionPadding left, RsaEncryptionPadding right)
 		{
-			if (ReferenceEquals (left, null))
-				return ReferenceEquals (right, null);
+			if (left is null)
+				return right is null;
 
```

Comment:
I like the shorthand, but does `is` have any different overhead vs ReferenceEquals?

Reply:
Machine code is identical for these.